pandas/tseries/holiday.py

```diff
@@ -394,8 +394,8 @@ def holidays(self, start=None, end=None, return_name=False):
         """
         if self.rules is None:
             raise Exception(
-                "Holiday Calendar {name} does not have any "
-                "rules specified".format(name=self.name)
+                f"Holiday Calendar {self.name} does not have any "
+                f"rules specified"
```

Comment:
f string prefix not needed here
```suggestion
                "rules specified"
```